lib/hooks/useUserRolesById/useUserRolesById.js

```diff
@@ -0,0 +1,44 @@
+import { useNamespace, useOkapiKy } from '@folio/stripes/core';
+import { useQuery } from 'react-query';
+
+/* Get list of user-roles by user IDs. That is, a list of roles assigned to the requested user IDs.
+// The response shape is:
+{
+  "userRoles": [
+    {
+        "userId": "<USER_ID>",
+        "roleId": "<ROLE_ID>",
+        "metadata": {
+            "createdDate": "<UTC_DATETIME>",
+            "createdBy": "<USER_ID>"
+        }
+    },
+    ...
+  ]
+}
+*/
+
+export function useUserRolesById(userId, tenantId) {
```

Comment:
I'd suggest providing the second argument as an object of options to keep it consistent and configurable

e.g.
https://github.com/folio-org/stripes-authorization-components/blob/master/lib/hooks/useApplicationCapabilities/useApplicationCapabilities.js#L28-L29
https://github.com/folio-org/stripes-authorization-components/blob/master/lib/hooks/useApplicationCapabilitySets/useApplicationCapabilitySets.js#L25-L26
https://github.com/folio-org/stripes-authorization-components/blob/master/lib/hooks/useAuthorizationPolicyById/useAuthorizationPolicyById.js#L9-L10
https://github.com/folio-org/stripes-authorization-components/blob/master/lib/hooks/useAuthorizationRoles/useAuthorizationRoles.js#L18-L19

Reply:
done